app/jobs/geocode_location_job.rb

```diff
@@ -47,6 +51,6 @@ def enqueue_next_job
   end
 
   def next_location
-    Location.where('latitude IS NULL OR longitude IS NULL').first
+    Location.where(geocode_response: nil).first
```

Comment:
👍 for using a constant. 

I don't think we could use `not null` because to select a non-geocoded location it uses `Location.where(geocode_response: nil).first`.